packages/liferay-changelog-generator/src/index.js

```diff
@@ -193,17 +193,59 @@ function linkToVersion(version, remote) {
 	}
 }
 
+/**
+ * Conventional commit types, and equivalent human-readable labels, in the order
+ * that we wish them to appear in the changelog.
+ *
+ * @see https://www.conventionalcommits.org/en/v1.0.0/
+ */
+const types = {
+	/* eslint-disable sort-keys */
+	feat: 'Features',
+	fix: 'Bug fixes',
+	perf: 'Peformance',
+	docs: 'Documentation',
```

Comment:
Might also want to be a bit fuzzy here because I think people might incorrectly use a "doc:" prefix at times.